### ISSUE TITLE
CURL error when launching the grabABI app with -s option (silent mode)

### ISSUE THREAD

**Issue by crodnun** [Great-Hill-Corporation/quickBlocks#217]
$ ./grabABI -s 0x1728039ce0d18a799c081c5c7fa2090dd365a8d0 
Reading ABI from EtherScan
Could not grab ABI for 0x1728039ce0d18a799c081c5c7fa2090dd365a8d0 from etherscan.io.
curl_easy_perform() failed: Couldn't connect to server

The help says this:

-s  (or --silent)     If ABI cannot be acquired, fail silently (useful for scripting)

But it seems to start a connection with a remote server somewhere.

**Comment by tjayrush** [comment]
This is a great bug, but I want to close it. The error message indicates something failed that, if the user wasn't warned about, wouldn't make any sense. The tool requests a file from a remote server, and depends on its return. If it doesn't get the file, it cannot really proceed.

The --silent means something slightly different, and I changed the code so that this option is still available for scripting, but no longer appears in the help text. I added a note to the README.md file just to be thorough and opened issue #238.
